### PR TITLE
fix/rounding for balance over hundred percent

### PR DESCRIPTION
**Pull request by sisco0**
**Before submitting this PR, please make sure**:

- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:

The changes presented in this PR are expected to fix the issue shown at https://github.com/hummingbot/hummingbot/issues/4946, which is caused by rounding errors at operations, as smart rounding could lead to lower values for the divisor of the operation which result is shown.

**Tests performed by the developer**:

The test is expected to be run by the issue creator as I could not replicate the scenario with my current configuration of the application. If there is any way to simulate the scenario without having wallets please let me know.

**Warning** These changes only could be merged after being tested by the issue creator or any other _HummingBot_ community member.

**Tips for QA testing**:

No additional recommendations are brought taking into account the slightness of the modifications.

### CODE REVIEW

**Comment by phbrgnomo** [comment]
Hello @sisco0 

A Pull Request Proposal was created related to this PR: https://snapshot.org/#/hbot.eth/proposal/0x54e13602869ce0e51447ad750a5b766ff64e142f24d6340434ef70e9a7e77cf4

Make sure to keep an eye on community reviews for the following days.

Thank you for contributing with Hummingbot Client!

File: hummingbot/client/command/balance_command.py [hunk 1]
```diff
@@ -108,10 +109,10 @@ async def show_balances(self):
             if df.empty:
                 self._notify("You have no balance on this exchange.")
             else:
-                lines = ["    " + line for line in df.to_string(index=False).split("\n")]
+                lines = ["    " + line for line in df.drop("sum_not_for_show", axis=1).to_string(index=False).split("\n")]
```
Comment (petioptrv) [inline hummingbot/client/command/balance_command.py:112]:
```suggestion
                lines = ["    " + line for line in df.drop(sum_not_for_show_name, axis=1).to_string(index=False).split("\n")]
```

**Review by petioptrv — APPROVED** [review]
Changes LGTM!

**Review by Nullably — APPROVED** [review]
LGTM

**Review by rapcmia — APPROVED** [review]
Tested and approved by QA
Test performed:
- Setup Pure market-making strategy
- Set the order amount high enough to use all assets of the set connector
- Execute balance command, all ok 
- Tested on source and docker build, all ok

**Pull request merged**